CMakeLists.txt

```diff
@@ -11,44 +11,31 @@ include(Functions.cmake)
 include(Testing.cmake)
 
 find_package(Threads REQUIRED)
-find_package(LLVM    REQUIRED CONFIG)
 find_package(Boost   REQUIRED COMPONENTS filesystem log program_options regex system thread)
 find_package(Odb     REQUIRED)
 find_package(Thrift  REQUIRED)
 find_package(Java    REQUIRED)
 find_package(GTest)
 
-# Check whether RTTI is on for LLVM
-message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
-message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
-message(STATUS "Using ClangConfig.cmake in: ${Clang_DIR}")
-message(STATUS "Using RTTI for LLVM: ${LLVM_ENABLE_RTTI}")
-if(NOT LLVM_ENABLE_RTTI)
-  message(SEND_ERROR "RTTI is required for LLVM")
-endif()
-
 include(UseJava)
 
 # Set the third-party libraries' path properly so the installation can find them.
 include(Exports.cmake)
 set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_RPATH};${RUNENV_LD_LIBRARY_PATH}")
 
+# Modules
```

Comment:
Is there any particular reason as to why the directories are in this order? If not, can we try having them in alphabetical ordering for ease of read?

Reply:
I only put the plugins directory to the end, because I think plugins should be added after all the core modules for a better CMake output and easier debugging in same cases. The order of these modules I guess is arbitrary.

Reply:
I reordered them, see https://github.com/Ericsson/CodeCompass/pull/343/commits/88acad41d4d1a23f6a7063fd1094ace85198e45f